src/assets/v1.3.0/compatibility_fallback_handler.json

```diff
@@ -27,6 +27,8 @@
     "1285": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "1287": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
     "4002": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "7341": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
+    "11437": "0xf48f2B2d2a534e402487b3ee7C18c33Aec0Fe5e4",
```

Comment:
Please make sure that this is ordered by chain id (numeric ascending)

Reply:
committed a fix for this